internal/env/config.go

```diff
@@ -21,35 +21,40 @@ var (
+	StringValues       string   `split_words:"true"`                        // Argument to pass to --set-string in applicable helm commands
+	ValuesFiles        []string `split_words:"true"`                        // Arguments to pass to --values in applicable helm commands
+	Namespace          string   ``                                          // Kubernetes namespace for all helm commands
+	KubeToken          string   `split_words:"true"`                        // Kubernetes authentication token to put in .kube/config
+	SkipTLSVerify      bool     `envconfig:"skip_tls_verify"`               // Put insecure-skip-tls-verify in .kube/config
+	Certificate        string   `envconfig:"kube_certificate"`              // The Kubernetes cluster CA's self-signed certificate (must be base64-encoded)
+	APIServer          string   `envconfig:"kube_api_server"`               // The Kubernetes cluster's API endpoint
+	ServiceAccount     string   `envconfig:"kube_service_account"`          // Account to use for connecting to the Kubernetes cluster
+	ChartVersion       string   `split_words:"true"`                        // Specific chart version to use in `helm upgrade`
+	DryRun             bool     `split_words:"true"`                        // Pass --dry-run to applicable helm commands
+	Wait               bool     `envconfig:"wait_for_upgrade"`              // Pass --wait to applicable helm commands
+	ReuseValues        bool     `split_words:"true"`                        // Pass --reuse-values to `helm upgrade`
+	KeepHistory        bool     `split_words:"true"`                        // Pass --keep-history to `helm uninstall`
+	Timeout            string   ``                                          // Argument to pass to --timeout in applicable helm commands
+	Chart              string   ``                                          // Chart argument to use in applicable helm commands
+	Release            string   ``                                          // Release argument to use in applicable helm commands
+	Force              bool     `envconfig:"force_upgrade"`                 // Pass --force to applicable helm commands
+	AtomicUpgrade      bool     `split_words:"true"`                        // Pass --atomic to `helm upgrade`
+	CleanupOnFail      bool     `envconfig:"cleanup_failed_upgrade"`        // Pass --cleanup-on-fail to `helm upgrade`
+	LintStrictly       bool     `split_words:"true"`                        // Pass --strict to `helm lint`
+	ConvertV2Releases  bool     `split_words:"true"`                        // Whether or not to use 2to3 convert to migrate Releases from v2 to v3
+	DeleteV2Releases   bool     `split_words:"true"`                        // Pass --delete-v2-releases option for 2to3 convert command
+	ReleaseVersionsMax int      `split_words:"true"`                        // Pass --release-versions-max option for 2to3 convert command
```

Comment:
No specific reason, we could set a default value here. I also noticed that using "default" in envconfig, hardcodes the value and it cannot be changed later, maybe is an issue in the version of envconfig used by the plugin or it's because how the config is handled in the plugin

Reply:
I set a default value of 10 if ReleaseVersionsMax is 0, that also means that a value of 0 is basically not allowed, but I think that's fine